scripts/dev/newReleaseBranch.py

```diff
@@ -0,0 +1,604 @@
+# - set acs.version.label comment to <!-- X.Y+1.0.<acs.version.label> -->
+# 4. community-share
+# - set scm-tag in main POM to HEAD
+# - set ACS version properties in main POM to X.Y+1.0
+# - set main POM versions to  X.Y+1.0.1-SNAPSHOT
+# 5. community-repo:
+# - set ACS version properties in main POM to X.Y+1.0
+# - set POM versions to  X.Y+1.0.1-SNAPSHOT
+# - set scm-tag in main POM to HEAD
+# - increment schema by 100 in repository.properties
+# - set version.revision to Z+1 in version.properties (test resources)
+# 6. acs-community-packaging
+# - set RELEASE_VERSION to X.Y+1.0-A1, DEVELOPMENT_VERSION to X.Y+1.0-A2-SNAPSHOT in ci.yml
+# - set POM versions to X.Y+1.0-SNAPSHOT
+# - set scm-tag in main POM to HEAD
+# - set comm-repo dependency in main POM to X.Y+1.0.1
+# - set comm-share dependency in main POM to X.Y+1.0.1
+#######################################
+# Update master branch for the next SP/major release
+# 1. acs-packaging:
+# - set RELEASE_VERSION to <next_development_version>-A1 passed as script argument or X.Y+1.0-A1 (if <next_development_version> not passed),
+#   DEVELOPMENT_VERSION to <next_development_version>-A2-SNAPSHOT or X.Y+1.0-A2-SNAPSHOT (if <next_development_version> not passed) in master_release.yml
+# - set POM versions to <next_development_version>-SNAPSHOT or X.Y+1.0-SNAPSHOT (if <next_development_version> not passed)
+# - set scm-tag in main POM to HEAD
+# 2. enterprise-share
+# - set scm-tag in main POM to HEAD
+# - set ACS version properties in main POM to <next_development_version> or X.Y+1.0 (if <next_development_version> not passed)
+# - set POM versions to <next_development_version>.1-SNAPSHOT or X.Y+1.0.1-SNAPSHOT (if <next_development_version> not passed)
+# 3. enterprise-repo:
+# - set POM versions to <next_development_version>.1-SNAPSHOT or X.Y+1.0.1-SNAPSHOT (if <next_development_version> not passed)
+# - set scm-tag in main POM to HEAD
+# - set acs.version.label comment to <!-- X.Y+1.0.<acs.version.label> -->
+# 4. community-share
+# - set scm-tag in main POM to HEAD
+# - set ACS version properties in main POM to <next_development_version> or X.Y+1.0 (if <next_development_version> not passed)
+# - set POM versions to <next_development_version>.1-SNAPSHOT or X.Y+1.0.1-SNAPSHOT (if <next_development_version> not passed)
+# 5. community-repo:
+# - set ACS version properties in main POM to <next_development_version> or X.Y+1.0 (if <next_development_version> not passed)
+# - set POM versions to <next_development_version>.1-SNAPSHOT or X.Y+1.0.1-SNAPSHOT (if <next_development_version> not passed)
+# - set scm-tag in main POM to HEAD
+# - increment schema by 100 (when next development minor version bumped) or 1000 (when next development version major bumped) in repository.properties
+# - set version.major/version.minor/version.revision to <next_development_version> or X.Y+1.0 (if <next_development_version> not passed) in version.properties (test resources)
+# 6. acs-community-packaging
```

Comment:
This part also performs version update in POM files (will include that in the doc comment)